src/manifolds/SymplecticStiefel.jl

```diff
@@ -452,9 +452,9 @@ That is, we find the element ``X ∈ T_p\mathrm{SpSt}(2n, 2k)``
 which solves the constrained optimization problem
 
 ````math
-    \operatorname{min}_{X ∈ ℝ^{2n×2k}} \frac{1}{2}||X - A||^2, \quad
+    \displazstyle\operatorname{min}_{X ∈ ℝ^{2n×2k}} \frac{1}{2}||X - A||^2, \quad
```

Comment:
This doesn't look correct.

Reply:
Indeed. German and Norwegian keyboards (also German and English) switch y and z so sometimes I still mess that up.

Reply:
Yes, it's annoying when this happens.

Reply:
Fixed it. So it was correct (in muscle movement, just for the wrong layout) ;)

Thanks for spotting this one! I tried checking all formulae I changed and must have missed that one.